### DISCUSSION TITLE
B站评论区改版，删除了页码，只能流式拉取而不能跳转到某一页了，不知能否通过脚本添加翻页功能？

### DISCUSSION THREAD

**Discussion by GalaxySnail** [the1812/Bilibili-Evolved#1862] [功能建议 / Ideas]
如题。这种流式拉取真的是非常智障的设计，拉很多以后（比如动态页）浏览器内存占用迅速上升最后OOM崩溃，导致事实上不可能访问很久以前的信息。这个改动直接把评论区从 seekable 改成 not seeakable 了。
说实话现在B站网页端做得真是越来越垃圾了，但符合叔叔的利益

**Comment by Geeyun-JY3** [comment]
重复：https://github.com/the1812/Bilibili-Evolved/discussions/1748#discussioncomment-640239
